src/components/RatingQuestion/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+
+const RatingQuestion = ({ initialRating, ratingCount, ratingEmoji }) => {
+  const [rating, setRating] = useState(initialRating);
+  const [emoji, setEmoji] = useState('👍🏻');
+  const [hover, setHover] = useState(null);
+  const emojiElement = [...Array(ratingCount || 5)];
+
+  useEffect(() => {
```

Comment:
Is it really necessary to use `useEffect` here since this the emoji type will not change once the component is rendered 🤔